collection-scripts/gather_monitoring

```diff
@@ -125,6 +126,11 @@ monitoring_gather(){
 
   alertmanager_get status  || true
 
+  # get some metrics by default
+  get_metrics_min_time
+  # TODO: Add out of the box matchers.
```

Comment:
Do you have any metrics in mind? maybe we can keep this for later?

Reply:
I don't but I don't look at must-gathers the way the CEE (Support team does; or the engineers supporting the bugs where they request this). We may want to start a survey or data collection request in some way to gather a list/plan to pull back diagnostic metrics.